.github/workflows/build.yml

```diff
@@ -14,14 +14,14 @@ env:
 
 jobs:
   build:
-    name: PHP ${{ matrix.php }} test on ${{ matrix.operating-system }} with ${{ matrix.dependencies }}
```

Comment:
Removed because this has not been a thing since we ceased supporting multiple versions of Symfony. So it was never actually being replaced with anything.